core/src/validator.rs

```diff
@@ -119,6 +119,51 @@ use {
 const MAX_COMPLETED_DATA_SETS_IN_CHANNEL: usize = 100_000;
 const WAIT_FOR_SUPERMAJORITY_THRESHOLD_PERCENT: u64 = 80;
 
+pub const DEFAULT_REPLAYING_BACKEND: &str = "blockstore-processor";
+pub const DEFAULT_BANKING_BACKEND: &str = "multi-iterator";
```

Comment:
I went with consistency of `XXX-ing` across this pr. (another discussion). i hate `BankingStage` <=> `ReplayStage` grammatical inconsistency....